pyproject.toml

```diff
@@ -52,6 +52,7 @@ packaging = "^21.3"
 psutil = "^5.9.4"
 jellyfish = "~0.10"
 altair = '>3.2, <5.0'
+pyvmomi = "^8.0.2.0.1"
```

Comment:
Hi @tgupta3. Thanks for the contribution! One question: we would like to keep the package as light as possible. Would it be possible for you to use the vcenter REST apis directly instead of the SDK? Look for example what we did in [netbox source](https://github.com/netenglabs/suzieq/blob/develop/suzieq/poller/controller/source/netbox.py)

Reply:
Yeah, I have tried using the REST API ([Commit](https://github.com/tgupta3/suzieq/blob/vcenter/suzieq/poller/controller/source/vcenter.py)) but it's slow. The REST API has the following steps
1. Get a list of all VMs
2. For each vm in the list, do another API to get the ip address. 

It's not common to have 500 VMs in a single vcenter deployment, and a REST API call sometimes takes 1-2 seconds. With the SOAP SDK, it can be done in a single API call, and takes a few seconds vs few minutes with REST API.

Reply:
Thanks @tgupta3 . Sorry for the late response. I see your point and it makes sense. Leave the code as is